### PR TITLE
adding The Good Docs Project

### PR DESCRIPTION
**Pull request by emckean**
thank you! 😄

## Your project

**1Password Teams url**: thegooddocsproject.1password.com (create the account before applying).

**Project name**:  The Good Docs Project

**Short description**: We're making templates to help open-source developers create better documentation!

**Project age**: 10 months

**Number of core contributors**: 5

**Project website**: thegooddocsproject.dev

**Repository url**: https://github.com/thegooddocsproject

**Latest release url**: https://github.com/thegooddocsproject/templates

**License type**: Zero-Clause BSD

**License url**: https://github.com/thegooddocsproject/templates/blob/master/LICENSE.txt

## Tell us about yourself

**Name**: Erin McKean

**Email**: emckean@google.com

**Project role**: steering committee member 

**Website**: erinmckean.com/about

### CODE REVIEW

**Comment by oliverdunk** [comment]
Hi @emckean! Thanks for bearing with us. I've gone ahead and applied the promotion to your account.

If you haven't done so already, I would definitely recommend implementing a recovery plan for your team in case access for a particular contributor is ever lost: https://support.1password.com/team-recovery-plan/. You are also welcome to add additional core contributors as you see fit.

Finally, do reach out to us in just under a year's time, as explained in the README. That way, we can look at continuing our support.

File: README.md [hunk 1]
```diff
@@ -615,4 +615,8 @@ https://webdriver.io
 
 ### HospitalRun
 Built with React, Node, PouchDB/CouchDB and using offline first design, we're working to deliver an HIS system that makes usability the #1 requirement, is built specifically for developing world requirements, and seeks to give back time to patient care. 
-https://hospitalrun.io
\ No newline at end of file
+https://hospitalrun.io
+
```
Comment (miwgmail) [inline README.md:619]: Ok

Comment (miwgmail) [inline README.md:619]: Good

File: README.md [hunk 2]
```diff
@@ -615,4 +615,8 @@ https://webdriver.io
 
 ### HospitalRun
 Built with React, Node, PouchDB/CouchDB and using offline first design, we're working to deliver an HIS system that makes usability the #1 requirement, is built specifically for developing world requirements, and seeks to give back time to patient care. 
-https://hospitalrun.io
\ No newline at end of file
+https://hospitalrun.io
```
Comment (miwgmail) [inline README.md:618]: ^^

File: README.md [hunk 3]
```diff
@@ -615,4 +615,8 @@ https://webdriver.io
...
+
+### The Good Docs Project
+We make templates to help developers create better documentation for open source projects.
+https://thegooddocsproject.dev
```
Comment (miwgmail) [inline README.md:622]: Ok

**Pull request merged**